.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -5,7 +5,7 @@ about: Suggest an idea for this project
 
 ### Is your feature request related to a problem? Please describe.
 A clear and concise description of what the problem is.
-Example. I'm am trying to do [...] but [...]
+Example: I am trying to do [...] but [...]
 
 ### Describe the solution you'd like
 A clear and concise description of what you want to happen.
```

Comment:
Personally I'd fold the `Describe alternatives you've considered` section into this one. Something like:

```suggestion
 A clear and concise description of what you want to happen. Include any alternative solutions you've considered.
```

Reply:
Go for it.

Reply:
Done!